lib/fakes3/file_store.rb

```diff
@@ -73,7 +73,7 @@ def create_bucket(bucket)
 
     def delete_bucket(bucket_name)
       bucket = get_bucket(bucket_name)
-      raise NoSuchBucket if !bucket
+      return unless bucket
```

Comment:
This could cause a discrepancy with how the real s3 works, but it seems like risk is low of us trying to do this in production so it's probably fine.